kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,104 @@ This sample demonstrates how to install the Traefik ingress controller to provid
-```
-$ curl -H 'host: traefik.example.com' http://${HOSTNAME}:30305/
-```
-
-## Optionally, download the Traefik Helm chart
-If you want, you can download the Traefik Helm chart and untar it into a local folder:
-```
-$ helm fetch  stable/traefik --untar
-```
-
-## Update the Traefik operator
-After the Traefik operator is installed and running, if you want to change some configurations of the operator, use `helm upgrade` to achieve this.
-```
-$ helm upgrade traefik-operator stable/traefik --values values.yaml 
+$ helm install traefik-operator traefik/traefik --namespace traefik --values values.yaml
 ```
 
-## Configure Traefik as a load balancer for WLS domains
-In this section we'll demonstrate how to use Traefik to handle traffic to backend WLS domains.
+## Configure Traefik as a load balancer for WebLogic domains
+This section describes how to use Traefik to handle traffic to backend WebLogic domains.
 
-### 1. Install WLS domains
-Now we need to prepare some domains for Traefik load balancing.
+### 1. Install WebLogic domains
+First, we need to prepare two domains for Traefik load balancing.
 
-Create two WLS domains:
-- One domain with name `domain1` under namespace `default`.
-- One domain with name `domain2` under namespace `test1`.
+Create two WebLogic domains:
+- One domain with `domain1` as domain uid under namespace `weblogic-domain1`.
+- One domain with `domain2` as domain uid under namespace `weblogic-domain2`.
 - Each domain has a web application installed with the URL context `testwebapp`.
+- Each domain has a WebLogic cluster `cluster-1` where each managed server listens on port `8001`.
 
-### 2. Install the Traefik Ingress
-#### Install a host-routing Ingress
+### 2. Web request routing 
+#### Host based routing 
+This sample demonstrates how to access application on two WebLogic domains using host based routing. Install a host-routing Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute)
 ```
 $ kubectl create -f samples/host-routing.yaml
+ingressroute.traefik.containo.us/traefik-hostrouting-1 created
+ingressroute.traefik.containo.us/traefik-hostrouting-2 created
 ```
-Now you can send requests to different WLS domains with the unique entry point of Traefik with different hostnames.
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different hostnames as defined in route section of `host-routing.yaml`.
 ```
-$ curl -H 'host: domain1.org' http://${HOSTNAME}:30305/testwebapp/
-$ curl -H 'host: domain2.org' http://${HOSTNAME}:30305/testwebapp/
+# Get the ingress controller web port
+$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')
+$ curl -H 'host: domain1.org' http://${HOSTNAME}:${LB_PORT}/testwebapp/
+$ curl -H 'host: domain2.org' http://${HOSTNAME}:${LB_PORT}/testwebapp/
 ```
-#### Install a path-routing Ingress
+
+#### Path based routing  
```

Comment:
Path based routing -> Path-based routing

Reply:
Modified to  Path-based routing